jwql/utils/preview_image.py

```diff
@@ -295,20 +300,21 @@ def make_figure(self, image, integration_number, min_value, max_value,
 
             # If making a thumbnail, make a figure with no axes
             if thumbnail:
-                fig = plt.imshow(shiftdata,
-                                 norm=colors.LogNorm(vmin=shiftmin,
-                                                     vmax=shiftmax),
-                                 cmap=self.cmap)
+                self.fig, ax = plt.subplots(figsize=(3, 3))
```

Comment:
Ha, this must be the `plt`/`fig` object mix up.

Reply:
Yeah, this was the crux of the problem with the bad thumbnails.